hashing/te-tag-query-python/TE.py

```diff
@@ -293,6 +293,79 @@ def getInfoForIDs(self, ids, **kwargs):
 
         return descriptors
 
+    # ----------------------------------------------------------------
+    # See also https://developers.facebook.com/docs/threat-exchange/reference/apis/threat-descriptors
+    #
+    # NOTE: THIS IS KNOWN TO NOT CORRECTLY HANDLE PAGINATION.  Please use the
+    # tagged_objects endpoint as detailed above.
+
```

Comment:
(comment, not blocking): We should this same warning to the public API docs on the top of https://developers.facebook.com/docs/threat-exchange/reference/apis/threat-descriptors

Reply:
Please sync with our colleague DQ as well :)